addons/account_edi_proxy_client/views/account_edi_proxy_user_views.xml

```diff
@@ -10,7 +10,6 @@
                         <group>
                             <group>
                                 <field name="company_id" groups="base.group_multi_company" invisible="1"/>
-                                <field name="edi_format_id" readonly="1" options="{'no_open': True}"/>
```

Comment:
Can we add the `proxy_type` ? If we add PEPPOL we'll have to tell one from the other